src/components/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// define the action type
```

Comment:
- [ ] Currently, your redux folder is inside the component folder which is not correct, its meant to be under `src` and not under components, I'll suggest you move the folder one level above. :smile: :+1:

Reply:
done, moved to source